File: Programmers/Level1/문자열내p와y의개수/문자열내p와y의개수.js
function solution(s) {
  const count = [0, 0];

  for (let i = 0; i < s.length; i++) {
    if (s[i].toUpperCase() === "P") {
      count[0] += 1;
    } else if (s[i].toUpperCase() === "Y") {
      count[1] += 1;
    }
  }
  return count[0] === count[1] ? true : false;
}

const s = "pPoooyY";

console.log(solution(s));

File: Programmers/Level1/문자열내p와y의개수/문자열내p와y의개수.py
def solution(s):
    answer = True
    p_cnt = 0
    y_cnt = 0
    for ch in s:
        if ch.upper() == 'P':
            p_cnt += 1
        elif ch.upper() == 'Y':
            y_cnt += 1
    if p_cnt == y_cnt:
        return True
    else:
        return False
    
s = "pPoooyY"
print(solution(s))